Metal properties added

BBarua opened the pull request:
1. Added Deformation models for 316H, 800H, A617, A282, and A230. 
2. Added Thermal properties for 316H, 800H, A617, A282, and A230. 
3. Added Damage properties for 316H, 800H, A617, A282, and A230. Used A617 and A740 D-diagram for A230 and A282, respectively.
3. Added Deformation model (based on Voce hardening rule) for A740 at T<1073.15K. Also, to be able to do interpolation of Voce hardening parameters between temperatures the Voce hardening parameters of T= 1073.15K and 1123.15K have been modified.

reverendbedford commented:
1. The A617.xml file in the data/thermal directory is misnamed (has quotes floating around in the file name).  Can you fix please?
2. Could you add a simple test that just tries to load each material we have available?  i.e. all it should do is 

library.load(whatever_mat, "base", "base", "base")

If that raises an exception then the test will fail.  This would catch (1) automatically in the future.

FYI all your new materials except A617 do work (A617 fails because of the filename issue).

BBarua commented:
Fixed A617.xml filename in data/thermal directory
Added 2 more temperatures in A282.xml in data/deformation directory
Modified A230.xml file in data/deformation directory
Added test_material_library.py in test directory
